src/main/java/au/org/emii/ncdfgenerator/CharValueEncoder.java

```diff
@@ -0,0 +1,56 @@
+package au.org.emii.ncdfgenerator;
+
+import ucar.ma2.Array;
+import ucar.ma2.DataType;
+
+import java.util.Map;
+
+class CharValueEncoder implements IValueEncoder {
+    private char fill;
+    private boolean haveFill;
+
+    CharValueEncoder() {
+        this.fill = ' ';
+        this.haveFill = false;
+    }
+
+    public DataType targetType() {
+        return DataType.CHAR;
+    }
+
+    public final void prepare(Map<String, Object> attributes) throws NcdfGeneratorException {
+        if (attributes.get("_FillValue") != null) {
+            try {
+                String s = (String)attributes.get("_FillValue");
+                fill = s.charAt(0);
+            }
+            catch (Exception e) {
+                throw new NcdfGeneratorException("Expected _FillValue attribute to be Char type");
+            }
+            haveFill = true;
+        }
```

Comment:
Should we be throwing an exception here, or is it OK to have no fillvalue if there are no missing values in which case this is not an exception